package.json

```diff
@@ -33,8 +34,8 @@
     "docsify-cli": "^4.1.10",
     "execa": "^0.8.0",
     "jest": "^21.2.1",
-    "jest-cli": "^21.2.1",
     "np": "^2.16.0",
+    "nyc": "^11.2.1",
```

Comment:
I think `nyc` is handled automatically isn't it by Jest?

Reply:
it is.  I don't know how this snuck back in.